modules/core/shared/src/main/scala/lucuma/core/instances/BoundedIntervalInstances.scala

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.instances
+
+import cats.Eq
+import spire.math.Bounded
+import spire.math.Interval
+
+trait BoundedIntervalInstances {
+  implicit def BoundedIntervalEq[A: Eq]: Eq[Bounded[A]] =
+    new Eq[Bounded[A]] {
+      def eqv(x: Bounded[A], y: Bounded[A]): Boolean = Interval.eq[A].eqv(x, y)
```

Comment:
Odd this wasn't defined in `spire` itself

Reply:
There's `Eq[Interval]` but not for its subclasses (and `Eq`'s type parameter is invariant).